apps/andi/lib/andi_web/controllers/dataset_page_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule AndiWeb.DatasetPageController do
```

Comment:
Not sure if I agree with calling this page `Dataset`. I was anticipating it being called `Status` or something like that. I mostly just want to avoid the proliferation of files with dataset in the name that we have in React. Were you imagining this controller would be responsible for other templates such as the dataset entry form as well?